website/templates/labels/png1/conf.php

```diff
@@ -1,20 +1,38 @@
 <?php
 
-$imgname = "$kret_szablon/label.jpg";
-$img = imagecreatefromjpeg($imgname); /* Attempt to open */
+require_once '../common.php';
+
+$imgname = "$kret_szablon/geokret_label_v1.png";
+$img = imagecreatefrompng($imgname); /* Attempt to open */
 
 $czarny = imagecolorallocate($img, 0, 0, 0);
-$czerwony = imagecolorallocate($img, 240, 0, 0);
+
+// Headers are Kuba Reczny 2005 font
 //$font = "../../fonts/cyberbit.ttf";
 //$font = "../fonts/techniczna.ttf";
 $font = '../fonts/lucida.ttf';
 
-imagettftext($img, 15, 0, 246, 44, $czarny, $font, "$kret_nazwa");
-imagettftext($img, 15, 0, 252, 122, $czarny, $font, $kret_owner);
-imagettftext($img, 10, 0, 252, 212, $czarny, $font, wordwrap(stripcslashes(strip_tags($kret_opis, '<img>')), 30));
+imagettftext($img, 35, 0, 600, 100, $czarny, $font, "$kret_nazwa");
+imagettftext($img, 35, 0, 600, 287, $czarny, $font, $kret_owner);
+imagettftext($img, 25, 0, 600, 500, $czarny, $font, mb_wordwrap(stripcslashes(strip_tags($kret_opis, '<img>')), 35));
+
+imagettftext($img, 35, 0, 1420, 100, $czarny, $font, $kret_tracking);
+imagettftext($img, 35, 0, 2010, 100, $czarny, $font, $kret_id);
+
+$manual_strings = array(
```

Comment:
@kumy did you know if [gettext](https://www.php.net/manual/fr/function.gettext.php) could enforce locale to use for a given translation ? 

thus in order to take advantage of the power of [crowdin](https://crowdin.com/project/geokrety)

ie having something like:

```` 
  'pl' => _($englishManualContent,'pl'),
  'en' => _($englishManualContent,'en'),
  'de' => _($englishManualContent,'de'),
  'cz' => _($englishManualContent,'cz'),
  'fr' => _($englishManualContent,'fr'),
  'ru' => _($englishManualContent,'ru')
```` 

(this feature seems not available, and switching locale is not a good way too, so Ignore this comment)

Reply:
This piece of code for sure should be improved to use proper localization strings and not hardcoded text. But so far seems none of templates actually use this "language" selector on the page and also I haven't seen any localization examples around, so we can keep it for later IMO